pkg/config/config.go

```diff
@@ -16,16 +16,16 @@ type imagesReference struct {
 	CloudControllerManagerOpenStack string `json:"cloudControllerManagerOpenStack"`
 }
 
-// operatorConfig contains configuration values for templating resources
-type operatorConfig struct {
+// Config contains configuration values for templating resources
+type Config struct {
```

Comment:
just curious, will we have any other config structs later? should we keep this named `OperatorConfig` ?

Reply:
Let's name it like this. Render will also create OperatorConfig, sounds ok.